queries.sql

```diff
@@ -1,8 +1,54 @@
+/*Queries that provide answers to the questions from all projects.*/
+
 SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE neutered = true;
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
 SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
-SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
-SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE name != 'Gabumon';
\ No newline at end of file
+SELECT name FROM animals WHERE neutered = true;
+SELECT * FROM animals WHERE name != 'Gabumon';
+SELECT name FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+
+/* Query to update and read the animals table data */
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name Like '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Based on the requirements, you are expected to have a query that will verify that a change was made. That query should be before and after the COMMIT as shown below 👇🏾 .

```sql
SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
```

Reply:
- [ ] @tanveerisonline   you haven't implemented the changes requested to have a query to verify changes made before and after the commit. Kindly add the query and also add screenshots for the same.